src/lib/src/auth/auth.component.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { ComponentFixture, TestBed, fakeAsync, tick, async } from '@angular/core/testing';
```

Comment:
fakeAsync and tick are unused imports (but they are a reason I haven't brought over the testing stuff from publish yet, doesn't support these)